src/views/basket.html

```diff
@@ -114,6 +114,16 @@ <h2 id="certificates-section-heading" class="govuk-heading-m">Certified certific
             rows: certificates
           }) }}
         {% endif %}
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-full">
+            {% if hasSameDayDelivery %}
+              <p class="govuk-body">Orders received before 11am will be sent out the same working day. Orders received after 11am will be sent out the next working day. We send UK orders by Royal Mail 1st Class post and international orders by Royal Mail International post.</p>
+            {% endif %}
+            {% if hasStandardDelivery %}
+              <p class="govuk-body">We will aim to send out the order within 10 working days. We send UK orders by Royal Mail 2nd Class post and international orders by Royal Mail International Standard post.</p>
```

Comment:
This line could be split into multiple lines to improve readability but not something that will stop this PR